garage/np/algos/base.py

```diff
@@ -1,7 +1,91 @@
-class Algorithm:
-    pass
+from collections import deque
 
+from dowel import tabular
+import numpy as np
 
-class RLAlgorithm(Algorithm):
-    def train(self):
-        raise NotImplementedError
+from garage.misc import special, tensor_utils
+
+
+class RLAlgorithm():
```

Comment:
this class is missing a docstring -- please add one

i'm surprised this passes the CI...